### PR TITLE
gitAndTools.git-dit: mark as broken

### PR DESCRIPTION
**Pull request by bhipple**
See inline comment; this does not build on Hydra and either requires custom
patching or a new release from upstream before it will build.

Note that we can still recompute the `cargoSha256` for #79975; the package
is broken in the identical way.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bhipple** [comment]
CC @Profpatsch @matthiasbeyer

**Review by Profpatsch — APPROVED** [review]

**Comment by Profpatsch** [comment]
I’d love to have this working, but I don’t have the capacity right now.

**Comment by matthiasbeyer** [comment]
You can package this, or wait until it is merged. This builds for me with rustc 1.40.0... so I suppose this could be packaged: https://github.com/neithernut/git-dit/pull/190

**Pull request merged**
